plugins/faustwp/faustwp.php

```diff
@@ -29,6 +29,47 @@
 define( 'FAUSTWP_PATH', plugin_basename( FAUSTWP_FILE ) );
 define( 'FAUSTWP_SLUG', dirname( plugin_basename( FAUSTWP_FILE ) ) );
 
+
+/**
+ * Get the minimum version of PHP required for this plugin.
+ *
+ * @return string Minimum version required.
+ */
+function faustwp_minimum_php_requirement() {
```

Comment:
I would love to suggest to have only one place for specifying the min php version - in plugin's header. That will allow us to edit min php version only once, and move the functionality into a standalone class in future. We can use `get_plugins` or `get_file_data` or `get_plugins_data`... But I am not sure how relevant it is now, maybe a good thing to keep in mind for future